### DISCUSSION TITLE
[EssentialsXDiscord Addon] Slash command to run a command on your linked minecraft account via Discord

### DISCUSSION THREAD

**Discussion by walllable** [EssentialsX/Essentials#4663] [Ideas and Feature Suggestions]
Hello! I'll get straight to the point as to how I'd like this to work, and explain my use-case for it after that.
I'd like it if we could link a Discord account to a Minecraft username/account (Seems this is already planned, so that helps!) and enter a slash command (Maybe `/playerexecute` or `/pe` as a fast alias) that executes a Minecraft command as the linked MC account, with the formatting of `/pe [command here]`

**Use-case:**
I've got Geyser running on Spigot server, so that a couple friends of mine using consoles can join. Entering commands via a console keyboard can be pretty slow, so I started looking for a solution that could make executing commands faster for them, and I ended up with two - Setting up premade commands in CommandPanels (and figuring out how to make those player-specific, and a way for me to add commands to a panel for them so they don't have to spend an eternity doing so with their keyboard,) or with `sudo`. I know that something like this can already by done with `/execute sudo [playernamehere] [command]` or `sudo [playernamehere] [command]` in a console relay channel, but I figure having to type `sudo` and specify the username every time could get pretty irritating, compared to it just being `/pe [command here]`.

**Comment by Chew** [comment]
Possibly being implemented in #4155 ?

**Reply by walllable** [reply]
I saw that already, but wasn't sure if it was better suited as its own issue like it is here, or as a comment on that PR.

**Reply by Chew** [reply]
Yeah def should go here, dunno if it's there or not yet but it would end up there as a commit if implemented 😄

**Reply by walllable** [reply]
👍
I found [this](https://github.com/orgs/EssentialsX/projects/1#card-57639460) earlier, and didn't see a feature like what I suggested, so I don't think it's planned, at least not yet. The other stuff already listed looks really cool, though. Looking forward to it.
